dts/bindings/sensor/adi,adxl345-common.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2020 Antmicro <www.antmicro.com>
+# SPDX-License-Identifier: Apache-2.0
+
+description: ADXL345 3-axis I2C accelerometer
+
+include: [sensor-device.yaml]
+
+properties:
+  frequency:
+    type: int
+    default: 100
+    description: |
+          Accelerometer sampling frequency (ODR). Default is 100Hz.
+    enum:
+      - 6
+      - 12
+      - 25
+      - 50
+      - 100
+      - 200
+      - 400
+
+  range:
+    type: int
+    default: 16
+    description: |
+      Accelerometer sampling range in G
```

Comment:
Need to justify the default value in the description. Is it a register reset value?

Reply:
Updated default value to reset value and updated description text.